### DISCUSSION TITLE
How to run in debug mode ?

### DISCUSSION THREAD

**Discussion by eszymko** [emqx/MQTTX#769] [Q&A]
How can we lunch mqttx in debug mode to view details logs when we cannot connect ?
The log tab in the app does not provide enough information

**Comment by ysfscream** [comment]
Debug mode is planned for a future release, currently, the log only provides simple step-by-step information.

Or do you have a better suggestion for a log message? When unable to connect, you can check the error message, or check the MQTT broker's logs. Usually, it may be that the MQTT broker.

**Reply by eszymko** [reply]
thanks for the answer. Detailed log during connexion would be great

**Reply by ysfscream** [reply]
Okay, for the refinement of the log we will design it in more detail.

**Comment by ysfscream** [comment]
Currently, we can select DEBUG logs in MQTTX Desktop to view more detailed connection activities. The CLI can set the --debug parameter to enable debug mode. Although this may not be the best implementation, it may be helpful for deeper debugging.
